cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -91,11 +91,32 @@
  */
 @SuppressWarnings({"checkstyle:ClassFanOutComplexity"})
 public class KafkaConnectCluster extends AbstractModel implements SupportsMetrics, SupportsLogging, SupportsJmx {
+     /**
+      * Default Strimzi Metrics Reporter allowlist for Kafka Connect.
+      * Check example dashboard compatibility in case of changes to existing regexes.
+      */
+    private static final List<String> DEFAULT_METRICS_ALLOW_LIST = List.of(
+            "kafka_admin_client_admin_client_metrics_connection_count",
+            "kafka_connect_connect_coordinator_metrics.*",
+            "kafka_connect_connector_metrics.*",
+            "kafka_connect_connector_task_metrics.*",
+            "kafka_connect_connect_worker_metrics_.*",
+            "kafka_connect_connect_worker_rebalance.*",
+            "kafka_connect_task_error_metrics.*",
+            "kafka_consumer_consumer_coordinator_metrics.*",
+            "kafka_consumer_consumer_fetch_manager_metrics.*",
+            "kafka_consumer_consumer_metrics.*",
+            "kafka_producer_producer_metrics.*",
+            "kafka_producer_producer_node_metrics_incoming_byte",
+            "kafka_producer_producer_topic.*"
+    );
+
     /**
-     * Default Strimzi Metrics Reporter allowlist.
+     * Default Strimzi Metrics Reporter allowlist for MirrorMaker 2.
+     * Includes both Kafka Connect metrics and MM2-specific metrics.
      * Check example dashboard compatibility in case of changes to existing regexes.
      */
-    private static final List<String> DEFAULT_METRICS_ALLOW_LIST = List.of(
+    protected static final List<String> DEFAULT_MIRROR_MAKER_2_METRICS_ALLOW_LIST = List.of(
```

Comment:
Two thoughts about this:
* Would it make sense to move this into the `KafkaMirrorMaker2Cluster` class, rename it to `DEFAULT_METRICS_ALLOW_LIST` to keep the things the same between the two classes?
* Should this _inherit_ from the Connect metrics? We could do something like this:
  ```java
  protected static final List<String> DEFAULT_MIRROR_MAKER_2_METRICS_ALLOW_LIST;
  static {
    DEFAULT_MIRROR_MAKER_2_METRICS_ALLOW_LIST = new ArrayList(KafkaConneCtcluster.DEFAULT_METRICS_ALLOW_LIST;
    DEFAULT_MIRROR_MAKER_2_METRICS_ALLOW_LIST.put("kafka_connect_mirror_mirrorcheckpointconnector.*");
    DEFAULT_MIRROR_MAKER_2_METRICS_ALLOW_LIST.put("kafka_connect_mirror_mirrorsourceconnector.*");
  }
  ```